astropy/io/fits/connect.py

```diff
@@ -127,7 +134,7 @@ def read_table_fits(input, hdu=None, astropy_native=False):
 
     else:
 
-        hdulist = fits_open(input)
+        hdulist = fits_open(input, character_as_bytes=True, memmap=memmap)
```

Comment:
Are you sure that `character_as_bytes` doesn't need to be settable in `Table.read()`?  E.g. someone does:
```
dat = Table.read('file.fits')
arr = dat.as_array()  # Don't like Table, use structured array
```
Then they are stuck with S-type arrays.

Reply:
@taldcroft - good point, I've made it configurable through Table.read.